src/test/scala/mesosphere/marathon/test/Mockito.scala

```diff
@@ -20,6 +20,7 @@ trait Mockito extends MockitoSugar {
   def times(num: Int) = M.times(num)
   def timeout(millis: Int) = M.timeout(millis.toLong)
   def atLeastOnce = M.atLeastOnce()
+  def once = M.times(1)
```

Comment:
I actually like it personally.

`verify(mock).abc() `vs `verify(mock, once).abc()` is much more obvious.